Tutorials/Sprite-Related-Tutorials/Sprite Layering Tutorial/01-02-sprites-layering.mdx

```diff
@@ -47,56 +69,118 @@ With our code at this stage, we basically just have a single-layered `Sprite`. U
 cases, but let's say we want to have user input, or specific functionality of the game "show" layers
 of a singular `Sprite`? Well, using our already-defined bitmaps, it's relatively simple.
 
+
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 SplashKit.SpriteAddLayer(bodySprite, hatBit, "hat");                        // layer 1
 SplashKit.SpriteAddLayer(bodySprite, starBit, "star");                      // layer 2
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+SplashKit.SpriteAddLayer(bodySprite, hatBit, "hat");                        // layer 1
+SplashKit.SpriteAddLayer(bodySprite, starBit, "star");                      // layer 2
+```
+</TabItem>
+</Tabs> 
 
 Using `SplashKit.SpriteAddLayer(Sprite, Bitmap, name)`, with our according parameters, we can add
 layers to our first `Sprite` object for later use. For reference, if you'd like to confirm that the
 layer indexes are correct, or see for yourself for debugging / testing purposes, the function
-`SpriteAddLayer()` has a return type of an integer, which corresponds to the assigned layer index
-value, which can be wrapped as:
+[SpriteLayerCount()](https://splashkit.io/api/sprites/#sprite-layer-count) has a return type of an integer, which can be wrapped as:
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 SplashKit.WriteLine(SplashKit.SpriteLayerCount(bodySprite));
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+write_line(sprite_layer_count(bodySprite));
+```
+</TabItem>
+</Tabs> 
+
+
+
 
 ## Showing and manipulating layers
 
 So now we have our created `Sprite`, our layers, and we can get to drawing our content to the
 screen, simple enough, right?
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 SplashKit.DrawSprite(bodySprite);
 spriteLayers.Refresh();
 SplashKit.Delay(2000);
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+draw_sprite(bodySprite);
+refresh_window(current_window());
+delay(1000);
+```
+</TabItem>
+</Tabs> 
+
 
 And, our layers aren't showing. It's important to know that layers aren't shown by default, so in
 order to show our added layers to our initial `Sprite` object, we call a function called
-`SplashKit.SpriteShowLayer(Sprite, idx);` with our parameters. For us, this looks something like:
+[ShowSpriteLayer()](https://splashkit.io/api/sprites/#sprite-show-layer) with our parameters. For us, this looks something like:
 
+<Tabs>
+  <TabItem label="C#">
 ```csharp
 SplashKit.SpriteShowLayer(bodySprite, 1);
 SplashKit.DrawSprite(bodySprite);
 spriteLayers.Refresh();
 SplashKit.Delay(2000);
 ```
+  </TabItem>
+  <TabItem label="C++">
+```c++
+sprite_show_layer(bodySprite, 1);
+clear_screen(COLOR_WHITE);
```

Comment:
Should the screen be cleared here? It isn't in the C# code.

Alternatively you could modify the C# code to clear the screen. Not sure what the differences might be.